lib/pbench/cli/agent/commands/cleanup.py

```diff
@@ -19,7 +18,7 @@ def execute(self):
             "%s deprecated, will be removed in future release in favor of pbench-clear-results",
             self.name,
         )
-        run_command("pbench-clear-results")
+        os.execvp("pbench-clear-results")
```

Comment:
So, `os.execlp("pbench-clear-results")` would have been better since you're not using an argv.

Reply:
Dang it, Peter!...I told you to change that!  🤪